src/app/manage-projects/manage-single-project/manage-single-project.component.ts

```diff
@@ -67,6 +67,7 @@ export class ManageSingleProjectComponent implements OnInit {
 
     createProjectWallet() {
         this.walletService.createProjectWalletTransaction(this.project.uuid).subscribe(res => {
+            SpinnerUtil.showSpinner();
```

Comment:
This way, after clicking the button to create a project wallet, there will be no spinner until our backend returns response of `this.walletService.createProjectWalletTransaction(this.project.uuid)`. This is an asynchronous method.
Set this line at the beginning of `createProjectWallet()` method (between lines 68 and 69).